python/ray/serve/task_processor.py

```diff
@@ -340,6 +332,11 @@ def __init__(self, config: TaskProcessorConfig):
 
         self._config = config
 
+        if RAY_SERVE_MAX_ONGOING_REQUESTS_ENV_KEY_INTERNAL in kwargs:
```

Comment:
from the POV of someone new writing their own TaskProcessor, they are not going to know the existence of `RAY_SERVE_MAX_ONGOING_REQUESTS_ENV_KEY_INTERNAL`. Can we change the initialize to
```
def initialize(self, max_ongoing_requests):
```

and call `initialize` from replica?